src/AddressSpace.cc

```diff
@@ -305,15 +305,15 @@ void AddressSpace::map_rr_page(AutoRemoteSyscalls& remote) {
     FATAL() << "Failed to locate " << fname;
   }
   path += fname;
-  size_t offset_pages = t->session().is_recording() ? 2 : 3;
+  size_t offset_pages = t->session().is_recording() ? 3 : 4;
```

Comment:
How about a constant that defines the number of pages in the increasingly-misnamed "rr page"?

Reply:
:+1: